Modelica/Magnetic/FundamentalWave/Components/Short.mo

```diff
@@ -1,9 +1,6 @@
 within Modelica.Magnetic.FundamentalWave.Components;
 model Short "Short cut branch"
   extends Magnetic.FundamentalWave.Interfaces.TwoPortElementary;
-equation
-  connect(port_p, port_n) annotation (Line(points={{-100,0},{-1,0},{-1,0},{
-          100,0}}, color={255,128,0}));
```

Comment:
See previous comment on short.

Reply:
Re-introduced in b39aaee0b04493cde0154b184e5c6242d2aa56c3